app/webpacker/controllers/flatpickr_controller.js

```diff
@@ -136,4 +143,12 @@ export default class extends Flatpickr {
         break;
     }
   };
+
+  setDefaultDateValue() {
+    if (this.defaultDateValue === "startOfDay") {
+      this.fp.setDate(moment().startOf("day").format());
+    } else if (this.defaultDateValue === "endOfDay") {
+      this.fp.setDate(moment().add(1, "days").startOf("day").format());
```

Comment:
This probably needs a small comment on that `23:59:00` vs. `23:59:59` thing ;)

Reply:
done : )